x-pack/plugins/synthetics/public/apps/synthetics/components/common/components/monitor_details_panel.tsx

```diff
@@ -262,8 +262,8 @@ const PROJECT_ID_LABEL = i18n.translate('xpack.synthetics.monitorList.projectIdH
   defaultMessage: 'Project ID',
 });
 
-const MONITOR_ID_ITEM_TEXT = i18n.translate('xpack.synthetics.monitorList.monitorIdItemText', {
-  defaultMessage: 'Monitor ID',
+const CONFIG_ID_ITEM_TEXT = i18n.translate('xpack.synthetics.monitorList.configIdItemText', {
```

Comment:
For a project monitor `Monitor ID` (queryable) makes more sense than `Config ID` as `config_id` could be volatile for a revived project monitor.

<img width="570" alt="Screenshot 2023-08-01 at 12 10 14" src="https://github.com/elastic/kibana/assets/2748376/8ee76e75-cb78-4f3b-b56e-e57fed91b363">